srcpkgs/keymapper/files/keymapperd/run

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/sh
```

Comment:
sorry i meant
```suggestion
#!/bin/sh
```

Reply:
That's what all other services have at least.

Reply:
Right, I should've caught that.